Library column polish

jonbell-lot23 opened the pull request:
A follow-up on this: https://github.com/RecordReplay/devtools/pull/3643/commits/ebb55155db34271ef1306dfc0cb0f39406658af8

Previously:
<img width="1167" alt="image" src="https://user-images.githubusercontent.com/9154902/133870052-d81b93fb-dcf3-4ed6-9787-54dfd1d84a1b.png">

Now:
<img width="1152" alt="image" src="https://user-images.githubusercontent.com/9154902/133870071-bcee5591-51b0-413a-84c8-640527f194f6.png">

jasonLaster commented:
Went back to tailwind for another reason and compared their stacked list with ours.

- wondering about using color as opposed to bold for the title
- wondering about the different color for the url
- do we want more vertical padding?
- could we use inter light in the footer?

<img width="1390" alt="Screen Shot 2021-09-18 at 5 47 37 AM" src="https://user-images.githubusercontent.com/254562/133889380-ad80c8fa-4829-4785-99d6-1c54dadf5300.png">

jonbell-lot23 commented:
**Color vs bold**
Our accent colour is blue, which looks like a link. We don't use accent colour anywhere in our visual language, so if we moved to that it would need to be part of a broader rethink of our design language.

**Different colour for URL**  
Not necessary, removed.

**More vertical padding?**  
A worthy exploration, but recommend another thread for that. This is a more short term fix.

**Inter light for second row**  
Added it to get the contrast we're looking for, and removed the bold header. We don't use light anywhere else in our UI, but it might be appropriate. To explore more.

**Avatars**  
They're pretty, but it doesn't solve enough of a need to offset the for the usability and IA issues it introduces.

Here's how it looks without bold, no color in header, and Inter light added to second row:
![image](https://user-images.githubusercontent.com/9154902/133913047-6d6895f6-5b88-49a7-9471-a725c93dfcf1.png)

Looks good.

jasonLaster commented:
Yep - light is good. 

Thanks for the explanation on color. With you on padding, avatars, ...
